integrations/twitter-v1.js

```diff
@@ -0,0 +1,101 @@
+try {
+    require('dotenv').config();
+} catch (ReferenceError) {
+    console.log('oh hey we must be running on Glitch');
+}
+const Twitter = require('twitter');
+const {addNewTwsfGuess} = require('../src/sqlite/twsf');
+
+const TOMUserId = '2827032970';
+const client = new Twitter({
+    consumer_key: process.env.TWITTER_API_KEY,
+    consumer_secret: process.env.TWITTER_API_KEY_SECRET,
+    access_token_key: process.env.TWITTER_ACCESS_TOKEN,
+    access_token_secret: process.env.TWITTER_ACCESS_TOKEN_SECRET,
+});
+
+const formatDirectMessage = async (event) => {
+    const {
+        id: twitterDmId,
+        message_create: {
+            sender_id: twitterId,
+            message_data: {text},
+        },
+    } = event;
+
+    const authorData = await client.get('users/show.json', {
+        user_id: twitterId,
+    });
+
+    const {name: twitterName, screen_name: twitterUsername} = authorData;
+
+    return {
+        author: {twitterId, twitterName, twitterUsername},
+        guess: {type: 'twitter dm', twitterDmId, text},
+    };
+};
+
+const fetchTwsfDirectMessages = async (cursor = undefined) => {
+    // Fetch direct message events
+    const {events} = await client.get('direct_messages/events/list.json', {
+        cursor,
+    });
+    console.log(`Found ${events.length} direct messsages.`);
+
+    // Filter messages. I'm not sure another type is ever provided, but can't
+    // hurt to be safe.
+    const directMessages = events.filter(
```

Comment:
merge all the filters
```js
event.type == 'message_create' && sender_id !== TOMUserId && ...
```
Do not hesitate to create a named function to make it pretty